ledger-service/fetch-contracts/src/main/scala/fetchcontracts/domain.scala

```diff
@@ -122,7 +122,7 @@ package domain {
             val payload = in.interfaceViews
               .find(_.interfaceId.exists(_ == id))
               .flatMap(_.viewValue) required "interfaceView"
-            (\/-(ContractTypeId.Interface fromLedgerApi id), None, payload)
```

Comment:
I think we can just use interfaceId here right?

Reply:
Yes.